scripts/build_dp_consensus.py

```diff
@@ -0,0 +1,154 @@
+import yaml
+from pathlib import Path
+from typing import Union
+
+import pandas as pd
+from pycytominer.operations import get_na_columns
+from pycytominer import consensus
+
+
+def infer_dp_features(dp_profile) -> list[str]:
```

Comment:
Is this what you mean by redundant code? There is no way to import it here?

Reply:
explained here: https://github.com/WayScience/CytoPipe/pull/17#discussion_r949587722